### PR TITLE
remove default value on empty storage item

### PR DESCRIPTION
**Pull request by khalillechelt**
@laszlokorte this PR should do the trick, wanna review?

### CODE REVIEW

**Comment by laszlokorte** [comment]
Yes that's what I have imagined.

File: src/responseCollection.js [hunk 1]
```diff
@@ -15,12 +15,17 @@ export default function(request$) {
...
-        const initialValue = localStorage.getItem(key) || ``
+        const initialValue = localStorage.getItem(key)
 
-        return local$
+        const value$ = local$
           .filter((req) => req.key === key)
           .map((req) => req.value)
-          .startWith(initialValue)
+
+        if (initialValue) {
```
Comment (laszlokorte) [inline src/responseCollection.js]: Maybe we should check for `initialValue!==null` explicitly so allow falsy values like `""` to be stored.

**Comment by khalillechelt** [comment]
added the type check

**Comment by laszlokorte** [comment]
Thank you

**Comment by khalillechelt** [comment]
no problem, will merge after the tests pass

should this be a major version bump or just minor?

**Comment by laszlokorte** [comment]
I think it should be a major again since somebody could rely on the empty item being emitted.

**Comment by khalillechelt** [comment]
yeah, I am thinking the same thing

**Pull request merged**